builtin/game/falling.lua

```diff
@@ -146,8 +146,22 @@ end
 
 local function drop_attached_node(p)
 	local n = core.get_node(p)
+	local drops = core.get_node_drops(n, "")
+	local def = core.registered_items[n.name]
+	if def and def.preserve_metadata then
+		local oldmeta = core.get_meta(p):to_table().fields
```

Comment:
isn't it possible that someone might want to preserve the inventory contents?

Reply:
Yes, I suppose that is possible.  We could leave off the oldmeta completely - they have the pos, the node has not been removed yet, the mod could fetch it for itself.